Sources/SRGDataProviderCombine/LiveCenterServices.swift

```diff
@@ -15,9 +15,12 @@ import Combine
 public extension SRGDataProvider {
     /**
      *  List of videos available from the Live Center.
+     *
+     *  - Parameter contentTypeFilter: The content type filter to apply.
+     *  - Parameter eventsWithResultOnly: Whether only medias which are in the Live Center results center must be returned. Related to sport events with a cesim id.
```

Comment:
_Related to sport events with a cesim id_ is probably too technical (I personally don't fully understand what this means).

Reply:
Thank you for the feedback. Cesim id is related to all SRGSSR sport api to trigger sport events.
As it's not exposed in the Integration Layer API, I removed the mention.